### PR TITLE
Updates for German and Croatian

### PR DESCRIPTION
**Pull request by milotype**
- updated/completed German and Croatian translations with newest .pot file
- some minor corrections in German
- changed the mention of "PDF-Shuffler" to "pdfarranger" in German
- remove out-dated lines in German

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -128,7 +164,23 @@ msgstr ""
 "%s ist ein Programm zum Umsortieren und Modifizieren von PDF-Dokumenten. "
 "Geschrieben in GTK+ und Python."
 
-#: pdfarranger/pdfarranger.py:1449
+#: pdfarranger/pdfarranger.py:1631
+msgid "Maintainers and contributors"
+msgstr "Verwalter und Mitwirkende"
```
Comment (dreua) [inline po/de.po:169]: I would prefer Maintainer here. 
https://de.m.wikipedia.org/wiki/Maintainer

File: po/de.po [hunk 2]
```diff
@@ -242,11 +290,11 @@ msgstr "Einfügen _vor Auswahl"
 
 #: data/menu.ui.h:19
 msgid "Paste Interleave _Odd"
-msgstr "Verzahnt einfügen _ungerade"
+msgstr "Leere _ungerade Seiten einfügen"
 
 #: data/menu.ui.h:20
 msgid "Paste Interleave _Even"
-msgstr "Verzahnt einfügen _gerade"
+msgstr "Leere _gerade Seiten einfügen"
```
Comment (dreua) [inline po/de.po:297]: Verzahnt is not great but with Leere I wouldn't have clue what this is doing.

File: po/de.po (hunk 1 above)
Comment (milotype) [reply po/de.po:169]: I don't see a reason to use English here. But "Verwalter" isn't nice, I agree.
Why not Entwickler or Hauptentwickler?

File: po/de.po (hunk 2 above)
Comment (milotype) [reply po/de.po:297]: As far as I understand it, it's an option to include even or odd empty pages.

File: po/de.po (hunk 1 above)
Comment (dreua) [reply po/de.po:169]: Hauptentwickler is a good idea! In context, I'd say going with "Entwickler" (plural) only would actually be the nicest.

Comment (milotype) [reply po/de.po:169]: OK, I'll change it to Entwickler.

File: po/de.po (hunk 2 above)
Comment (dreua) [reply po/de.po:297]: No, I'm not aware of a function to add empty pages at all. This is about pasting the pages from clipboard in every second place. This is useful for example if you scanned all the fronts of your pages first and than all the backs and you want to put it back in a PDF in a way where each backside comes after it's front side.

Pages in Clipboard: A B C D
Pages displayed: 1 2 3 4
After paste interleave odd: 1 A 2 B 3 C 4 D

Comment (milotype) [reply po/de.po:297]: Sorry, I misinterpreted it then. Thank you for explaining.
How about **Verschachtelt** instead of Verzahnt?

Comment (milotype) [reply po/de.po:297]: Two more alternatives:
**Alternierend** or **Abwechselnd**?

Comment (dreua) [reply po/de.po:297]: I think Verzahnt is better than Verschachtelt.
I'm okay with Alternierend, Abwechselnd or Verzahnt, not sure what is best.

Comment (milotype) [reply po/de.po:297]: In that case I'd propose **Abwechselnd**.
But if you want, you may decide ;-)

**Comment by milotype** [comment]
@dreua 
I've made the necessary corrections … files could be merged now …

**Comment by jeromerobert** [comment]
Merged: 85227fd. Thanks.

**Pull request closed without merging**